mlperf_logging/rcp_checker/README.md

```diff
@@ -2,10 +2,28 @@
 
 MLPerf reference convergence point (RCP) checker
 
+## Description
+
+Currently this ingests RCP points from json file and provides API
+to run convergence checks using t-tests, find convergence points
+and generate interpolated convergence points.
+
 ## Usage
 
-TBD
+From the mlperf_logging directory:
+
+python3 -m rcp_checker
+
+Currently this loads RCPs from 1.0.0/rcps.json and runs a basic test
+
```

Comment:
```suggestion

RCP checker is called by result_summarizer.
Verifying your submission with result_summarizer is enough,
it doesn't have to be checked with rcp_checker separately.

```

Reply:
Yes, this needs to be updated anyway, the plan it to be able to validate a directory standalone, and the whole submission from the result summarizer and the package checker.